dataclients/kubernetes/httpsredirect_test.go

```diff
@@ -321,19 +319,19 @@ func TestEnableHTTPSRedirectFromIngress(t *testing.T) {
 			-> "http://1.2.3.4:8080";
 		kube_namespace1__ingress1__www_example_org___foo__service1_https_redirect:
 			Header("X-Forwarded-Proto", "http") &&
-			HeaderRegexp("X-Forwarded-Port", ".*") &&
 			Host("^www[.]example[.]org$") &&
 			PathRegexp("^/foo") &&
+			PathRegexp(".*") &&
```

Comment:
please add a comment here, nobody will know why this extra regex is here (needs explanation!)